west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8d93a161567dcc525771f70692587e6a37947b71
+      revision: pull/1565/head
```

Comment:
Please do not update revision here, see https://github.com/nrfconnect/sdk-nrf/pull/14408

Reply:
Err just did it...

Reply:
Can this PR just be closed?